### PR TITLE
(feat):support load more dataset with different formats

### PR DESCRIPTION
**Pull request by jinronga**
**Description**

This PR feature  https://github.com/SkywardAI/chat-backend/issues/95

There are code irregularities，

The function has been implemented, but the code is not standardized and will be optimized in the future.

**[Signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: backend/src/repository/rag/chat.py [hunk 1]
```diff
@@ -61,12 +62,39 @@ async def load_csv_file(self, file_name: str, model_name: str) -> bool:
...
+           await self.load_data_set_by_field(param=param)   
+        return True
+
+
+
+    async def load_data_set_all_field(self, dataset_name: str)-> bool:
+        print(f"load_data_set_all_field dataset_name:{dataset_name}")
         reader_dataset=load_dataset(dataset_name)
         for item_dict in reader_dataset['train']:
+            print(f"load_data_set_all_field item_dict:{item_dict.get('url', '')},{type(item_dict)}")
```
Comment (Micost) [inline backend/src/repository/rag/chat.py:79]: bettter use log instead of print

**Review by Micost — APPROVED** [review]
LGTM

**Review by Micost — DISMISSED** [review]
LGTM

**Comment by Micost** [comment]
@jinronga please solve the conflict

**Review by Micost — APPROVED** [review]
Lgtm

**Comment by Aisuko** [comment]
The CI issue is the secret key permission issue. I am trying to fix that. So, we can merge the PR. Thanks for your guys @jinronga @Micost. And I will also invite @jinronga to the org.

**Review by Aisuko — APPROVED** [review]
approved

**Pull request merged**